packages/ledger.rb

```diff
@@ -0,0 +1,35 @@
+require 'package'
+
+class Ledger < Package
+  description 'A double-entry accounting system with a command-line reporting interface'
+  homepage 'https://www.ledger-cli.org/'
+  version '3.1.3'
+  source_url 'https://github.com/ledger/ledger/archive/v3.1.3.tar.gz'
+  source_sha256 'b248c91d65c7a101b9d6226025f2b4bf3dabe94c0c49ab6d51ce84a22a39622b'
+
+  binary_url ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+  binary_sha256 ({
+    aarch64: '',
+     armv7l: '',
+       i686: '',
+     x86_64: '',
+  })
+
+  depends_on 'gmp' => :build
+  depends_on 'boost' => :build
+  depends_on 'mpfr' => :build
+
+  def self.build
+    system "./acprep"
+    system 'make'
```

Comment:
We will probably want to add `prefix=#{CREW_PREFIX}` and possibly `libdir=#{CREW_LIB_PREFIX}` to abstract the install location.

Reply:
Option --prefix=#{CREW_PREFIX} is OK. But this package don't provide --libdir option. Is okay that just add --prefix option?

Reply:
If no `--libdir` configure option is available, that usually indicates there are no library files included in this app.  It is fine to use only `--prefix`.